docs/schema/0-Schema.mdx

```diff
@@ -11,73 +11,91 @@ import JSONSchemaViewer from "@theme/JSONSchemaViewer";
+
+<JSONSchemaViewer
+  schema={ yamlLoad(MainDefs) }
+  resolverOptions={ generateResolverOptions( {yamlBasePath: '/', jsonPointer:"#/$defs/propertyContainers/overtureFeaturePropertiesContainer" })}/>
 
-### Core schema properties
+<!-- Below is an example of how you can reference just 1 property within the properties container -->
+<!-- <JSONSchemaViewer
+  schema={ yamlLoad(MainDefs) }
+  resolverOptions={ generateResolverOptions( {yamlBasePath: '/', jsonPointer:"#/$defs/propertyContainers/overtureFeaturePropertiesContainer/properties/sources" })}/> -->
 
-Every feature in Overture has a core set of properties that are described in the schema. Overture features:
+ 
+### Schema concepts
+This is where I'll briefly mention the concepts that are important to understand, such as linear referencing, topology, cross-theme releationships, contested borders, internationalization, mapping OpenStreetMap tags to Overture properties, and more. This will link to the concepts section of this documentation. 
 
-- have a type
-- have a geometry, where the type of geometry is constrained by the feature type
-- are strongly-typed, _i.e._ the feature type constrains the geometry and properties
-- have properties, which may include a core set of "flat" properties and additional properties with a nested structure
-- have an ID property which is globally unique within the ID-space of the entire Overture data distribution version. For some feature types, the ID is registered with GERS
-- may have custom user extension properties
+### Schema conventions
+These are the styles, notations, nomenclatures, specifications, and standards we have adopted for the Overture schema. 
 
-### Schema notation conventions
+#### Notations
 
 - snake case is used for all property names, string enumeration members, and string-valued enumeration equivalents
 - boolean properties have a prefix verb "is" or "has" in a way that grammatically makes sense
       e.g.
   - `has_street_lights=true`
   - `is_accessible=false`
 
-### Measurements
+#### Measurements
 
 Measurements of real-world objects and features follow [The International System of Units (SI)](https://www.bipm.org/en/publications/si-brochure): heights, widths, lengths, etc. In the Overture schema, these values are provided as scalar numeric value without units such as feet or meters. Overture does this to maximize consistency and predictability.
 
 Quantities specified in regulatory rules, norms and customs follow local specifications wherever possible. In the schema, these values are provided as two-element arrays where the first element is the scalar numeric value and the second value is the units. Overture uses local units of measurement -- feet in the United States and meters in the EU, for example. The exact unit is confirmed in the specification of the property but is not repeated in the data.
 
-### Relations
+#### Relations
 
 Low-cardinality directed relations are stored as ID references on the source feature.
```

Comment:
I hadn't caught this earlier, but can we change this to "Relationships"?

I'm trying to eliminate the use of the term "Relation" because of the confusion it creates with the OSM NWR data model Relation entity. Also because I would argue that a relation and a relationship are not the same.

Reference: https://github.com/OvertureMaps/schema-wg/blob/admin/decisions/adr-relationship-modeling.md

> The term relat*ion* should not be used to mean relation*ship*. The term relat*ion* should only be used for describing OpenStreetMap relations and abstractions that are equivalent to OSM relations.